src/stores/draw.store.ts

```diff
@@ -0,0 +1,64 @@
+import { defineStore } from 'pinia'
+import { reactive } from 'vue'
+
+export type DrawState = {
+  drawPointActive: boolean
+  drawLabelActive: boolean
+  drawLineActive: boolean
+  drawCircleActive: boolean
+  drawPolygonActive: boolean
+}
+
+export const useDrawStore = defineStore('draw', () => {
+  const drawState = reactive({
+    drawPointActive: false,
+    drawLabelActive: false,
+    drawLineActive: false,
+    drawCircleActive: false,
+    drawPolygonActive: false,
+  } as DrawState)
```

Comment:
I don't know if this is storing an object is very useful here, I would have stored a ref : string containing the active drawing tool key. But maybe I am missing something. This is not a big deal, we can keep it this way if you prefer.

Reply:
Ah, good point, I've adapted the code to your suggestion. The templates just get a little more verbose this way, but it's a lot less code indeed. Is this what you had in mind?